camp/ocampbell/XBox Trigger Project/XBox Trigger Project/.gitignore

```diff
@@ -0,0 +1,172 @@
+# This gitignore has been specially created by the WPILib team.
```

Comment:
Why do you need two folders named 'XBox Trigger Project'?

Also no spaces in path names